### ISSUE TITLE
Blacklist Suggestion

### ISSUE THREAD

**Issue by Fumeknight79** [Sivelos1/SivsItemsRoR2#16]
So I was playing some rebar MUL-T with swarms and vengeance on, and got into a sniping match with my clones. I had the Abandoned Wisp item. Was kinda even, with my bad aim and my wisps getting in my vision, but I found out my clones were creating wisps of their own by killing mine with either misses or meat hook procs. This ended up creating a loop, and I realized I had to go up close and kill them quick. A grenade was able to kill all of the wisps, and the gasoline blasts killed my clones, but I ended up spawning enough little wisps to crash the game. 

TL;DR: umbras shouldn't have abandoned wisps, or you could inadvertently crash your game with wisp spam.

Also, I don't know if it's intentional or not, but maybe make the mob items command-proof like aurelionite items, or aspects from zetaspects? Seems kinda busted to just get them from a random yellow drop from a boss, and it seems kinda op to get a boss item from a random mob, too. And you can buy them from bigger bazaar.
